tests/test_iotedgedev.py

```diff
@@ -481,27 +481,32 @@ def test_validate_deployment_template_and_manifest_failed():
         else:
             result = runner_invoke(['genconfig', '-f', 'deployment.template_invalidresult.json'])
 
+        template_path = os.path.join(tests_assets_dir, "deployment.template_invalidresult.json")
```

Comment:
Seems like you should put this file name in a const or make it configurable.

Reply:
Declared the file name as local variable, in case the test case need to use another test file in the future.